src/views/PageSearchPortal.vue

```diff
@@ -85,11 +91,22 @@ export default {
         ]),
         async getParkingRequests() {
             this.isLoading = true;
-            const parkingRequestList = await mayaClient.get(
-                '/internal/parking-requests',
-            );
-            this.parkingRequests = parkingRequestList;
-            this.isLoading = false;
+            const mobile = this.VOMobile;
+            if (mobile === mobile) {
+                const mobileNo = await mayaClient.get(
```

Comment:
please rename the variable. As 'mobileNo' suggests it will have mobile number. But its story parkingRequests filtered by mobile number.